git_machete/exceptions.py

```diff
@@ -1,9 +1,16 @@
+import sys
+
 from git_machete import utils
 
 
 class MacheteException(Exception):
```

Comment:
Also! back to the original issue:

![image](https://user-images.githubusercontent.com/3383210/130253165-ae1f47e9-0538-4ebc-a778-42880cfe4290.png)

I'd rather expect just `Not a git repository` error message... that's also how git-machete v3.3.0 behaves:

![image](https://user-images.githubusercontent.com/3383210/130253285-3da111b6-80fb-4165-8732-0053ad49abbc.png)

Reply:
whoops, solution was so unexpectedly simple, just move initialisation of MacheteClient couple line down to try clause.